### DISCUSSION TITLE
Parameters for smooth movement.

### DISCUSSION THREAD

**Discussion by ascoider** [asus4/tf-lite-unity-sample#287] [Q&A]
Hi!
I was wondering, which variables and parameters are the ones that affects the most to the smoothness of the joint detection and movement, but avoiding as less delay as possible.

Let's say I want to move an object by positioning it in over a tracked hand. I already checked "Use filter" and "filter scalability" and I can see the difference (in PC, lower number = smooth but delayed. in android it's the opposite, i don't know why).
Are there more variables in your code for this purpose? If not, in your opinion, what would you recommend to add to the tracking of the joint to make it smoother? I already tried a Lerp in the moving object but it makes it creates too much delay.

Thank u very much for your responses.

**Comment by asus4** [comment]
Hello @ascoider, 

I utilized the "Relative Velocity Filter" algorithm. You can find more information about the parameters by accessing the [original Media Pipe code](https://github.com/google/mediapipe/blob/master/mediapipe/util/filtering/relative_velocity_filter.h).

However, I concur that the filter parameters are heavily reliant on the frame rate. I'm uncertain which parameter would be optimal. Additionally, the latest Media Pipe repository utilizes the "one euro filter" algorithm by default, which appears to be more straightforward and responsive. I plan to implement it when I have some free time.
